src/modules/ordersTable/pure/ReceiptModal/index.tsx

```diff
@@ -82,6 +88,12 @@ export function ReceiptModal({
           <CloseIcon onClick={() => onDismiss()} />
         </styledEl.Header>
 
+        {twapOrder && (
+          <styledEl.InfoBannerWrapper>
+            <InlineBanner type="information" content={`Part of a ${twapOrder.order.n}-part TWAP order split`} />
```

Comment:
Could we mention which part this is?

Like: `Part 5 of a 10 part TWAP...`

Reply:
Ohhhh.
Then it's not a part of. It is THE order itself

Reply:
I'm confused you, sorry. Then the text should be like:
```
TWAP order split into 5 parts
```

I will rework it

Reply:
Parent:
<img width="485" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/05783e65-3598-46c4-a359-6b504304f5c3">

Children:
<img width="487" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/abf364ee-63e9-4fd2-acee-b95e1faa2e88">

>Could we mention which part this is?

Unfortunately, we can not right now. It's a part of the task: https://github.com/cowprotocol/cowswap/issues/2640

Reply:
Fixed:
https://github.com/cowprotocol/cowswap/pull/2652/commits/07316dbd8c0cce8055066d2215e2daf2bf10031e